main/src/library/Assert.flix

```diff
@@ -31,6 +31,30 @@ mod Assert {
 
     use RichString.{blue, red, text, yellow};
     import java.lang.AssertionError;
+    use Assert.FnInfo.FnInfo
+    use Assert.SourceLocation.SourceLocation
+    use Assert.SourcePosition.SourcePosition
+    pub enum SourcePosition({lineOneIndexed = Int32, colOneIndexed = Int32})
+    pub enum SourceLocation({source = String, sp1 = SourcePosition, sp2= SourcePosition})
```

Comment:
We should put these elsewhere. The question is of course where. Then the smart constructors should go in their companion.